### PR TITLE
Local module for AMPcombi database downloading

### LINKED ISSUE #238: AMPcombi DRAMP database download failing for many samples when running in pipeline
### Description of the bug

All but one connections gets blocked, causing all other AMPcombi runs to fail when running via e.g. tower or AWS megatests etc.

### Command used and terminal output

_No response_

### Relevant files

_No response_

### System information

_No response_

### PR DESCRIPTION
**Pull request by jfy133**
Closes #238 

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/funcscan/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/funcscan _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: subworkflows/local/amp.nf [hunk 1]
```diff
@@ -78,14 +79,23 @@ workflow AMP {
 
     //AMPCOMBI
     ch_input_for_ampcombi = ch_ampresults_for_ampcombi
+        .dump(tag: "ampcombi_raw")
```
Comment (jfy133) [inline subworkflows/local/amp.nf:82]:
```suggestion
```

File: subworkflows/local/amp.nf [hunk 2]
```diff
@@ -78,14 +79,23 @@ workflow AMP {
...
         .groupTuple()
-        .join( ch_faa_for_ampcombi )
+        .dump(tag: "ampcombi_group")
```
Comment (jfy133) [inline subworkflows/local/amp.nf:84]:
```suggestion
```

File: subworkflows/local/amp.nf [hunk 3]
```diff
@@ -78,14 +79,23 @@ workflow AMP {
...
+        .join( ch_faa_for_ampcombi.dump(tag: "faa") )
```
Comment (jfy133) [inline subworkflows/local/amp.nf:85]:
```suggestion
        .join( ch_faa_for_ampcombi )
```

File: subworkflows/local/amp.nf [hunk 4]
```diff
@@ -78,14 +79,23 @@ workflow AMP {
...
+        .dump(tag: "ampcombi_join")
```
Comment (jfy133) [inline subworkflows/local/amp.nf:86]:
```suggestion
```

**Review by louperelo — APPROVED** [review]
Looks good to me.

**Comment by jfy133** [comment]
If you are happy to fix the merge conflicts you're welcome to do it and merge :) otherwise I can do it next week

**Pull request merged**